src/main/java/org/tron/core/db2/core/SnapshotManager.java

```diff
@@ -40,6 +40,7 @@
   private static final int DEFAULT_STACK_MAX_SIZE = 256;
   public static final int DEFAULT_MAX_FLUSH_COUNT = 500;
   public static final int DEFAULT_MIN_FLUSH_COUNT = 1;
+  public LevelDbDataSourceImpl tmplevelDbDataSource;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make tmplevelDbDataSource a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1760&issues=AWc1aJ6TDUHak5Vv3ejb&open=AWc1aJ6TDUHak5Vv3ejb)

[ISSUE_KEY]: AWc1aJ6TDUHak5Vv3ejb